### DISCUSSION TITLE
Safari vs Chrome

### DISCUSSION THREAD

**Discussion by RubenReuring** [idiotWu/smooth-scrollbar#498] [Q&A]
Safari smooth scroll is way 'slower' compared to Chrome. I have this issue on desktop aswell as mobile. On mobile the slowness is so bad I almost want to disable the smooth scrollbar. Is there anything I could do to improve this scrolling on Safari?

**Comment by dolphin-wood** [comment]
I also noticed this issue, and the weirdest thing is that once you switch to another tab and then switch back to the smooth scrollbar, all the laggy animations will go away. 🤔

**Reply by RubenReuring** [reply]
Wow that's right. I hadn't even noticed that.

On Mac Safari it's still somewhat doable, but mobile Safari is terrible. Currently I have controlled the scroll speed with the plugin system, where Chrome gets a lower speed than Safari, but Safari mobile is still too slow. Is there any way to go above '1' speed with the plugin system as well?

**Reply by dolphin-wood** [reply]
@RubenReuring you can try this plugin to scale the scrolling speed: https://github.com/idiotWu/smooth-scrollbar/blob/develop/docs/migration.md#optionsspeed, but I doubt if it will work since the rendering frame rate appears much lower than 60fps.
